ci_scripts/ci_post_clone.sh

```diff
@@ -1,9 +1,71 @@
 #!/bin/sh
 
+set -o pipefail
+
 # Set the -e flag to stop running the script in case a command returns
 # a non-zero exit code.
 set -e
 
-if [ "$CI_WORKFLOW" = "SwiftLint" ]; then
-    brew install swiftlint
-fi
+# Print env
+env
+
+# Create CoreSimulator.log
+touch ~/Library/Logs/CoreSimulator/CoreSimulator.log || true
```

Comment:
We shouldn't need to do any of the stuff with CoreSimulator.log any more. Switching between simulator versions used to be incredibly buggy and we needed the logs to figure out what was breaking, but thankfully that's been obsolete for years.